File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.c
#include <stdio.h>

int main(void){
    int a, b, c;
    
    while (1)
    {
        scanf("%d %d %d", &a, &b, &c);
        if (a == 0 && b == 0 && c == 0) break;
        
        if (a > b && a > c && a >= b + c) {
            printf("Invalid\n"); 
            continue;
        }
        if (b > a && b > c && b >= a + c) {
            printf("Invalid\n"); 
            continue;
        }
        if (c > a && c > b && c >= a + b) {
            printf("Invalid\n"); 
            continue;
        }
        
        if (a == b && b == c) printf("Equilateral\n");
        else if (a == b || b == c || a == c) printf("Isosceles\n");
        else if (a != b && b != c && a != c) printf("Scalene\n");
    }
    return 0;
}

File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.py
while True :
    sides = list(map(int, input().split()))
    if (sides[0] == 0 and sides[1] == 0 and sides[2] == 0) :
        break
        
    if (max(sides) < sum(sides) - max(sides)) :
        if (sides[0] == sides[1] and sides[1] == sides[2]) :
            print("Equilateral")
        elif (sides[0] == sides[1] or sides[1] == sides[2] or sides[2] == sides[0]) :
            print("Isosceles")
        else :
            print("Scalene")
    else :
        print("Invalid")